packages/inventory/src/ducks/inventory.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  fetchActionCreatorFactory,
+  removeActionCreatorFactory,
+  setTotalRecordsFactory,
+  reducerFactory,
+  getItemsByIdFactory,
+  getItemsArrayFactory,
+  getItemByIdFactory,
+  getTotalRecordsFactory,
+} from '@opensrp/reducer-factory';
+
+/** interface inventory */
+export interface Inventory {
+  deliveryDate: string;
+  donor: string;
+  poNumber: number;
+  productName: string;
+  providerId: string;
+  quantity: number;
+  serialNumber: string;
+  servicePointId: string;
+  stockId: string;
+  unicefSection: string;
+}
+
+/** reducer name */
+export const reducerName = 'inventory';
+
+/** Action types */
+const customfetchedActionType = 'inventory/INVENTORIES_FETCHED';
+const customRemoveActionType = 'inventory/REMOVE_INVENTORIES';
+const customSetTotalRecordsActionType = 'inventory/SET_TOTAL_INVENTORIES';
```

Comment:
I have often had problems with this custom action types, so until that is fixed, you can remove this

Reply:
https://github.com/OpenSRP/web/issues/190